[2024-11-14T21:10:47.891+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-14T21:10:47.901+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2024-11-14T11:28:00+00:00 [queued]>
[2024-11-14T21:10:47.908+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2024-11-14T11:28:00+00:00 [queued]>
[2024-11-14T21:10:47.908+0900] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-11-14T21:10:47.923+0900] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-11-14 11:28:00+00:00
[2024-11-14T21:10:47.925+0900] {standard_task_runner.py:72} INFO - Started process 2378042 to run task
[2024-11-14T21:10:47.928+0900] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'first_task', 'scheduled__2024-11-14T11:28:00+00:00', '--job-id', '287', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp1z7h5ag3']
[2024-11-14T21:10:47.930+0900] {standard_task_runner.py:105} INFO - Job 287: Subtask first_task
[2024-11-14T21:10:47.963+0900] {task_command.py:467} INFO - Running <TaskInstance: first_dag.first_task scheduled__2024-11-14T11:28:00+00:00 [running]> on host MN
[2024-11-14T21:10:48.031+0900] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T11:28:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-14T11:28:00+00:00'
[2024-11-14T21:10:48.032+0900] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-14T21:10:48.047+0900] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-14T21:10:48.047+0900] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the fist task!']
[2024-11-14T21:10:48.052+0900] {subprocess.py:99} INFO - Output:
[2024-11-14T21:10:48.053+0900] {subprocess.py:106} INFO - hello world, this is the fist task!
[2024-11-14T21:10:48.054+0900] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-14T21:10:48.072+0900] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-14T21:10:48.072+0900] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=first_task, run_id=scheduled__2024-11-14T11:28:00+00:00, execution_date=20241114T112800, start_date=20241114T121047, end_date=20241114T121048
[2024-11-14T21:10:48.100+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-14T21:10:48.112+0900] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T21:10:48.113+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
